src/components/Walk/ResultsPage.js

```diff
@@ -2,22 +2,120 @@ import React from "react";
 import Typography from '@mui/material/Typography';
 import Container from '@mui/material/Container';
 import Grid from '@mui/material/Grid';
+import QuestionGraph from "../Graphs/QuestionGraph";
+import WalkGraph from "../Graphs/WalkGraph";
+
+/**
+ * GRAPH 1: Questions Data format
+ */
+const questionAnswers = [
+    // Dictionary of questions in the order in which the questions were asked.
+    {
+        question: "Question 1?",
+        answers: [
+            {
+                "answer": "YES", // answer option
+                "count": 2  // Number of users who chose this answer for this question. 
+            },
+            {
+                "answer": "NO",
+                "count": 3
+            }
+        ],  
+    },
+    {
+        question: "Question 2?",
+        answers: [
+            {
+                "answer": "High Scool's",
+                "count": 10
+            },
+            {
+                "answer": "Bachelor's",
+                "count": 4
+            },
+            {
+                "answer": "Masters",
+                "count": 3
+            }
+        ],      
+    }
+]
+
+
+
+
+/*
+GRAPH 2: 
+Expected API data format for both host and Participant results page.
```

Comment:
2nd Graph will contain data about the perspective walk graph. It will show the position of the different users. 
Same data format will be sent to both host and participant. The only difference is value in "participantLocation". This is what will differentiate how data is displayed between different users.